mne/decoding/time_gen.py

```diff
@@ -352,6 +359,18 @@ def is_regressor(clf):
                                    'epochs to score()')
 
         # Check scorer
+        if self.score_mode not in ('per-fold', 'mean-across-folds', 'no-cv'):
+            raise ValueError("`score_mode` must be 'per-fold', "
+                             "'mean-across-folds' or 'no-cv'. "
+                             "Got %s instead'" % self.score_mode)
+        score_mode = self.score_mode
+        if (
+            (self.predict_mode == 'mean-prediction') and
+            (self.score_mode != 'no-cv')
+        ):
```

Comment:
funny way of writing this. maybe we can conventionalize this a bit., remove inner brackets and reduce number of lines.